src/parser/github-comment-module.ts

```diff
@@ -23,6 +23,7 @@ interface SortedTasks {
 export class GithubCommentModule implements Module {
   private readonly _configuration: GithubCommentConfiguration = configuration.incentives.githubComment;
   private readonly _debugFilePath = "./output.html";
+  private _lastCommentId: number | null = process.env.COMMENT_ID ? Number(process.env.COMMENT_ID) : null;
```

Comment:
This is not super clear that `process.env.COMMENT_ID` is taken from [compute.yml](https://github.com/ubiquibot/conversation-rewards/pull/52/files#diff-786f3dda14e85b9321efd6100c191f3da5b32c5411f6aa3c43f0f20b9a7bb363R52) and should not be set manually but I can't make up a more explicit solution

Reply:
Will add a comment describing what this does to make it more clear.

Reply:
Let me know if that help
https://github.com/ubiquibot/conversation-rewards/pull/52/commits/866f853a0bf7078cb943da71cc039f07fcf9bf0d#diff-b4df32515a0c65918d3eb9741db81ca0afd56b41270db64491d774d9edab7a28R26